app/models/ucas_match.rb

```diff
@@ -30,6 +31,8 @@ def trackable_applicant_key
   end
 
   def action_needed?
+    return false if manually_resolved?
```

Comment:
Could we do
```
  def resolved?
    %w[resolved_on_apply resolved_on_ucas manually_resolved].include?(action_taken)
  end
```
instead of adding a clause here.